caikit_computer_vision/data_model/object_detection.py

```diff
@@ -44,10 +42,10 @@ class Point2d(DataObjectBase):
 
 @dataobject(package="caikit_data_model.caikit_computer_vision")
 class BoundingBox(DataObjectBase):
-    xmin: Annotated[int, FieldNumber(1)]
-    xmax: Annotated[int, FieldNumber(2)]
-    ymin: Annotated[int, FieldNumber(3)]
-    ymax: Annotated[int, FieldNumber(4)]
+    xmin: Annotated[float, FieldNumber(1)]
+    xmax: Annotated[float, FieldNumber(2)]
+    ymin: Annotated[float, FieldNumber(3)]
+    ymax: Annotated[float, FieldNumber(4)]
```

Comment:
Just a note that the existing object detector should be updated to give float coords to be consistent - I can make this change in a separate PR